src/stactools/sentinel2/commands.py

```diff
@@ -55,11 +56,15 @@ def create_item_command(
             with open(providers) as f:
                 additional_providers = json.load(f)
 
+        antimeridian_strategy = "split"
+        strategy = Strategy[antimeridian_strategy.upper()]
```

Comment:
Worth exposing the antimeridian strategy as an argument or option to the command?